tests/libvmi/cloudinit.go

```diff
@@ -55,19 +55,23 @@ func WithCloudInitNoCloudNetworkData(data string) Option {
 }
 
 // WithCloudInitConfigDriveData adds cloud-init config-drive user data.
-func WithCloudInitConfigDriveData(data string, b64Encoding bool) Option {
+func WithCloudInitConfigDriveData(userData, networkData string) Option {
 	return func(vmi *v1.VirtualMachineInstance) {
 		addDiskVolumeWithCloudInitConfigDrive(vmi, cloudInitDiskName, v1.DiskBusVirtio)
 
 		volume := getVolume(vmi, cloudInitDiskName)
-		if b64Encoding {
-			encodedData := base64.StdEncoding.EncodeToString([]byte(data))
-			volume.CloudInitConfigDrive.UserData = ""
-			volume.CloudInitConfigDrive.UserDataBase64 = encodedData
-		} else {
-			volume.CloudInitConfigDrive.UserData = data
-			volume.CloudInitConfigDrive.UserDataBase64 = ""
-		}
+		volume.CloudInitConfigDrive.UserData = userData
+		volume.CloudInitConfigDrive.NetworkData = networkData
+	}
+}
+
+// WithCloudInitConfigDriveDataBase64 adds cloud-init config-drive with base64 encoded user data.
+func WithCloudInitConfigDriveDataBase64(userData string) Option {
```

Comment:
Is this used somewhere?

Reply:
No.
The base64 userdata was not tested before this PR.

Reply:
Drop it if it's not used?

Reply:
Added a commit to remove it.